### PR TITLE
Merge appsembler release

### PR DESCRIPTION
**Pull request by melvinsoft**
This PR is primary for merge azure backups into eucalyptus. Is the follow up the last PR for merge `appsembler/master` into `appsembler/release`. Now we are merging `appsembler/release` into `appsembler/eucalyptus/develop`.

### CODE REVIEW

**Comment by melvinsoft** [comment]
[(2) Enable backups on Azure for Eucalyptus deployments](https://trello.com/c/Sg1Ynwth/958-2-enable-backups-on-azure-for-eucalyptus-deployments)

File: playbooks/amc.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@
     - role: mongo
       when: "'localhost' in EDXAPP_MONGO_HOSTS"
     - { role: 'rabbitmq', rabbitmq_ip: '127.0.0.1' }
+    # Do we need need edxapp 2 times here????
```
Comment (tkeemon) [inline playbooks/amc.yml]: Yes, you do need the two `edxapp` roles there. The first sets up the Celery works, the second sets up the main LMS/CMS processes.

File: playbooks/testdrive.yml [hunk 2]
```diff
@@ -36,12 +36,13 @@
     - edxlocal
     - role: mongo
       when: "'localhost' in EDXAPP_MONGO_HOSTS"
+    # Do we relly need edxapp 2x times here?
```
Comment (tkeemon) [inline playbooks/testdrive.yml]: Yes, see my comments in amc.yml.

File: playbooks/testdrive.yml [hunk 3]
```diff
@@ -36,12 +36,13 @@
...
     - { role: 'edxapp', celery_worker: True }
     - edxapp
-    - notifier
     - analytics_api
     - insights
     - edx_notes_api
+    # Do we really need this 2x times?
```
Comment (tkeemon) [inline playbooks/testdrive.yml]: I think so. My impression is that the first `demo` role installs the default Open edX course that's installed with every site deployment. And the second `demo` role is the Appsembler branded demo course. 

@bezidejni might be the best person to ask about this one.

**Comment by denibertovic** [comment]
LGTM.

File: playbooks/amc.yml (hunk 1 above)
Comment (johnbaldwin) [inline playbooks/amc.yml]: This would be good to note here. I can see this question popping up again in the future.

**Pull request merged**